login.php

```diff
@@ -79,7 +79,7 @@
 		</div>
 		<!--Div for the image -->
 		<div class="img_2">
-			<img src = "images/img2.jpg">
+			<img src = "images/music-1106439_1280.jpg">
```

Comment:
22/08/2021: I trailing for some img to be in the login page so it can make the page look less empty and display a great aesthetic so it can received  user satisfaction(first impression).